helper_image/scripts/copy-out.sh

```diff
@@ -1,48 +1,85 @@
 #!/bin/bash
 
+set -e
+
 function usage() {
     echo
     echo "Verify the md5sums of two files match"
     echo "This is used after copy_out directives to ensure the copy_in file matches the copy_out file"
     echo
-    echo "Syntax: copy-out.sh FILE_IN FILE_OUT"
+    echo "Syntax: copy-out.sh FILE_IN FILE_OUT <NUM_COMPUTES>"
+    echo
+    echo "Set <NUM_COMPUTES> to 0 when index mount directories are not expected"
     echo
     echo "example:"
-    echo "copy-out.sh /lus/global/testuser/test.in /lus/global/testuser/test.out"
+    echo "copy-out.sh /lus/global/testuser/test.in /lus/global/testuser/test.out 0"
+    echo "copy-out.sh /lus/global/testuser/test.in /lus/global/testuser/*/test.out 4"
     echo
 }
 
 COPY_IN=$1
 COPY_OUT=$2
+NUM_COMPUTES=$3
 
 if [[ -z "$COPY_IN" ]]; then
     usage
     exit 1
 elif [[ -z "$COPY_OUT" ]]; then
     usage
     exit 1
+elif [[ -z "$NUM_COMPUTES" ]]; then
```

Comment:
run shellcheck, and don't quote unless it flags this.
The ``[[`` test doesn't usually need quotes; it's built-in, unlike the old-style `[` test.